CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to the Conjur Katacoda Scenarios
+
+For general contribution and community guidelines, please see the [community repo](https://github.com/cyberark/community).
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Katacoda Documentation](#katacoda-documentation)
+- [Building and Testing](#building-and-testing)
+- [Troubleshooting Steps](#troubleshooting-steps)
+- [Contribution Workflow](#contribution-workflow)
+
+## Prerequisites
+The Katacoda CLI is recommended in order to create a new scenario. You can install the Katacoda CLI by running `npm install katacoda-cli --global` in the terminal.
+
+## Katacoda Documentation
+A link to the Katacoda documentation can be found [here](https://www.katacoda.community/welcome.html).
+
+Additionally, it is recommended that you look at the other scenarios in the `cyberark/katacoda-scenarios` to see how Katacoda has been used in other tutorials.
+
+## Building and Testing
+The Katacoda scenarios can be tested using the `katacoda-cli`. In order to validate that the structure of your scenario is properly defined, you can run `katacoda validate:all .` in the root directory of the `cyberark/katacoda-scenarios` directory.
```

Comment:
Link to repo needs to be the full link

Reply:
Is the standard to link the repository anytime you are referring to it?